components/eam/src/chemistry/utils/modal_aero_calcsize.F90

```diff
@@ -513,16 +513,20 @@ subroutine modal_aero_calcsize_sub(state, deltat, pbuf, ptend, do_adjust_in, &
    !-----------------------------------------------------------------------
 
    ! arguments
-   type(physics_state), target, intent(in)    :: state       ! Physics state variables
-   type(physics_buffer_desc),   pointer       :: pbuf(:)     ! physics buffer
+   integer,                     intent(in)    :: ncol
+   integer,                     intent(in)    :: lchnk
+   real(r8),                    intent(in)    :: pdel(:,:)
+   real(r8),                    intent(in)    :: state_q(:,:,:) 
    real(r8),                    intent(in)    :: deltat      ! model time-step size (s)
+   type(ptr2d_t),               intent(inout) :: qqcw(:)     ! Cloud borne aerosols mixing ratios [kg/kg or 1/kg]
    type(physics_ptend), target, optional, intent(inout) :: ptend       ! indivdual parameterization tendencies
 
    logical,  optional, intent(in) :: do_adjust_in
    logical,  optional, intent(in) :: do_aitacc_transfer_in
    logical,  optional, intent(in) :: update_mmr_in
    integer,  optional, intent(in) :: list_idx_in       ! diagnostic list index
-   real(r8), optional, intent(inout), allocatable, target :: dgnumdry_m(:,:,:) ! interstital aerosol dry number mode radius (m)
+   !real(r8), optional, intent(inout), allocatable, target :: dgnumdry_m(:,:,:) ! interstital aerosol dry number mode radius (m)
```

Comment:
The above commented line can be removed?

Reply:
Removed